examples/aggregates/README.md

```diff
@@ -0,0 +1,60 @@
+# Astarte Aggregates example
+
+This example will guide you through setting up a simple ESP32 application running the Astarte SDK
+and interacting with a remote instance of Astarte.
+
+The example device to perform the following actions:
+- Connect to a remote Astarte instance using a manually generated Device ID and credential secret.
+- When new aggregated object values are published on a server owned interface, then the device will
+publish new aggregated object values on the device owned interface.
+
+If you want to start using a local instance of Astarte hosted on your machine and register the
+device automatically you can check out the [`toggle_led` example](../toggle_led) in this repository.
+
+We will assume you already performed the common configuration and satisfy the common prerequisites.
+
+## Device connection to Astarte
+
+Once the example has been flashed it will automatically attempt to connect the device to the remote
+instance of Astarte.
+Verify this by using the monitor functionality of the ESP-IDF to observe the log messages of the
+ESP32.
+After connection has been accomplished, the following ESP32 log message should be shown:
+```
+ASTARTE_EXAMPLE_AGGREGATE: Astarte device connected, session_present: 0
+```
+You can see how the connection has been achieved also from the Astarte Dashboard by opening the
+newly created device and observing the status that should have changed to *Connected*.
+
+## Send and receive data from the device
+
+Now that our device is connected to Astarte is time to start publishing data.
+We can interact with the remote Astarte instance using the `astartectl` tool.
+
+We will use two commands in this example, `send-data` and `get-samples`.
+With `send-data` we can publish new values for a server owned interface.
+The syntax is the following:
+```
+astartectl --astarte-url "<API_URL>" --token "<TOKEN>" appengine --realm-name "<REALM_NAME>" \
+    devices send-data <DEVICE_NAME>                                                          \
+    org.astarteplatform.esp32.examples.ServerAggregate /11                                   \
+    '{"longinteger_endpoint":45993543534, "booleanarray_endpoint":[false,false,true,true]}'
+```
+Where `<TOKEN>` is the shell variable containing the token, `<REALM_NAME>` is your realm's name and
+`<API_URL>` is the API's URL as found on astarte cloud console. `<DEVICE_NAME>` is the device ID to
```

Comment:
I believe we can drop saxon genitives in this context (_realm name_, _API URL_), but up to you.